### ISSUE TITLE
0.3.0 Breaks support for API Keys and Secrets

### ISSUE THREAD

**Issue by timcole** [UrielCh/proxmox-api#9]
Upgrading from 0.2.0 to 0.3.0 causes every request to throw `Error: login failed with 401: authentication failure` when authenticating using tokenId and tokenSecret.

Downgrading back to 0.2.0 works.

```
[cause]: Error: Auth https://{REDACTED}:8006/api2/json/access/ticket Failed! with Exception: Error: login failed with 401: authentication failure
```

**Comment by UrielCh** [comment]
I'm sorry about that, can you provide me a code sample to reproduce?

Any calls using token secret? get the issue.

**Comment by timcole** [comment]
Literally any calls yeah. The "Using Api token" example at the bottom of the readme doesn't even work anymore.

**Comment by UrielCh** [comment]
regression reproduced.

**Comment by UrielCh** [comment]
regression fixed, v0.3.1 published.

**Comment by timcole** [comment]
0.3.1 fixed this issue, thanks 👍🏻
